scan/git.go

```diff
@@ -150,7 +150,14 @@ func gitLfs(root string) error {
 func checkoutGit(root, ref, subdir string) (string, error) {
 	// Try checking out by ref name first. This will work on branches and sets
 	// .git/HEAD to the current branch name
-	if output, err := gitWithinDir(root, "checkout", ref); err != nil {
+	// If the reference format is "pull/{pull-request-number}/head", then checkout to
```

Comment:
if the repo has multiple pull requests, which one does the code checkout?

Reply:
Can you add a comments to clarify for pull request reference, previous step has already fetched it explicitly and current step just needs to check out the head.

Reply:
Added